pages/_app.tsx

```diff
@@ -1,9 +1,42 @@
 import 'normalize.css/normalize.css';
 import '../global.scss';
 import '../icomoon-v1.0/style.css';
-
+import { DefaultSeo } from 'next-seo';
 import type { AppProps } from 'next/app';
 
+const meta = {
+  title: 'Polski Front-End',
+  description: `Serwis Polski Front-End powstał w celu zebrania w jednym miejscu jak największej liczby stron, serwisów oraz blogów na temat szeroko rozumianego front-end developmentu. Co ważne, wszystkie zgromadzone tutaj serwisy tworzone są w języku polskim!`,
+};
+
 export default function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />;
+  return (
+    <>
+      <DefaultSeo
+        title={meta.title}
```

Comment:
Można użyć tego ich title template i zawsze dodawać | Polski Frontend

Reply:
IMO jest to gorsza opcja, lepiej by homepage był po prostu `Polski Front-End` a nie np. `Home | Polski Front-End`

Reply:
Dając template `| Polski Frontend`, tak będzie wyglądać domyślnie tytuł

Reply:
A nie można titleTemplate ustawić na null tylko na stronie głównej albo coś

Reply:
Trzeba by było zrobić tak, że gdy nie podany jest title do layoutu, to ustawia template na null. Imo mija się to z celem, by robić takie akrobacje tylko by skorzystać z tej jednej funkcji

Reply:
Title musi być wymaganym propsem, więc ten problem znika :P A ustawić na null tylko na stronie głównej.